### DISCUSSION TITLE
QR Code Plugin

### DISCUSSION THREAD

**Discussion by eurobank** [mindstellar/shopclass#473] [Q&A] [answered]
I do need the QR Code and it doesn't even install (fatal errors) with Osclass.

Anyone has fixed this plugin?

**Comment by navjottomer** [comment, marked as answer]
Where is the source code?

**Reply by eurobank** [reply]
This is from the RIP osclass org market

[1568153228-qrcode_103.zip](https://github.com/mindstellar/Osclass/files/7845215/1568153228-qrcode_103.zip)

**Reply by navjottomer** [reply]
I have uploaded the version here: https://github.com/navjottomer/qrcode
I didn't get any issue installing it on dev version PHP 7.4, PHP 8.0
I update you when I test it more.

**Comment by eurobank** [comment]
It seems to install and work fine.

Thank you, i really needed this.

**Reply by eurobank** [reply]
But doesn't save changes ....
